ipa-core/src/test_fixture/world.rs

```diff
@@ -190,6 +200,13 @@ impl Default for TestWorld {
     }
 }
 
+impl TestWorld {
+    #[must_use]
+    pub fn with_seed(seed: u64) -> Self {
```

Comment:
Default values for generic parameters is a blessing and a curse.

`impl TestWorld<NotSharded>` that exists on the line 266 is probably a better place for this fn.

Reply:
Is there a reason to implement `Default` only for `TestWorld<NotSharded>`?

Reply:
backward compatibility was the only one - I didn't want to change a bunch of non-sharded tests when introducing the in-memory sharding infrastructure